autofit/non_linear/fitness.py

```diff
@@ -1,7 +1,23 @@
+import logging
 import numpy as np
 
+from autoconf import conf
+
 from autofit import exc
 
+from timeout_decorator import timeout
+
+def get_timeout_seconds():
+
+    try:
+        return conf.instance["general"]["test"]["lh_timeout_seconds"]
+    except KeyError:
+        pass

```

Comment:
Catch the key error at the point of use rather than having to add a None-check

Reply:
`get_timeout_seconds()` is used twice in the code, so catching the error at the point of use would lead to duplicaiton.